docs/drivers.md

```diff
@@ -22,6 +22,15 @@ To install the KVM2 driver, first install and configure the prereqs:
 * Ubuntu 16.x or older: `sudo apt install libvirt-bin libvirt-daemon-system qemu-kvm`
 * Fedora/CentOS/RHEL: `sudo yum install libvirt-daemon-kvm qemu-kvm`
 
+```
+Enable,start, and verify the libvirtd service has started. 
+```shell
+systemctl enable libvirtd.service
```

Comment:
Normally one wants to use "sudo" here, to avoid that annoying popup

Reply:
good suggestion. I found that i needed to be root to run through the documentation . If that's the case then sudo is not needed.  I updated the doc to include sudo just in case